src/samples/msal-b2c-web-sample/src/main/java/com/microsoft/azure/msalwebsample/JSONHelper.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.azure.msalwebsample;
+
+import java.lang.reflect.Field;
+
+import org.apache.commons.lang3.StringUtils;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+class JSONHelper {
```

Comment:
Looks like this is not being used anywhere, can be deleted.

Reply:
ok